frontend/dockerfile/docs/syntax.md

```diff
@@ -323,7 +323,10 @@ RUN FOO=abc ash /app/script.sh
 
 * `BUILDKIT_CACHE_MOUNT_NS=<string>` set optional cache ID namespace
 * `BUILDKIT_CONTEXT_KEEP_GIT_DIR=<bool>` trigger git context to keep the `.git` directory
-* `BUILDKIT_INLINE_CACHE=<bool>` inline cache metadata to image configuration or not (for Docker-integrated BuildKit (`DOCKER_BUILDKIT=1 docker build`) and `docker buildx`)
+* `BUILDKIT_INLINE_BUILDINFO_ATTRS=<bool>`¹ inline build info attributes in image config or not
```

Comment:
Where is this implemented?

Reply:
In the [image](https://github.com/crazy-max/buildkit/blob/597b8c9d3817468a6915057cbd3b27e9971c9e78/exporter/containerimage/export.go#L185-L194) and [oci](https://github.com/crazy-max/buildkit/blob/597b8c9d3817468a6915057cbd3b27e9971c9e78/exporter/oci/export.go#L123-L132) exporters via https://github.com/docker/buildx/pull/908/files (should also be done in moby when vendored).

Reply:
Ah, yes. But it is kind of weird to document it in here as it is a buildx client feature.

Reply:
Yeah that's the same for `BUILDKIT_INLINE_CACHE` in the README and https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#built-in-build-args.